nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/schema/FlowFileSchema.java

```diff
@@ -64,4 +65,23 @@
 
         FLOWFILE_SCHEMA_V1 = new RecordSchema(flowFileFields);
     }
+
+    static {
```

Comment:
I think the same comment as below applies here as well - we can have a private static method that takes an argument of type FieldType and builds this RecordSchema, rather than duplicating all of this code.